presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1562,6 +1562,38 @@ public void testMaterializedViewForUnionAll()
         }
     }
 
+    @Test
+    public void testMaterializedViewForExcept()
+    {
+        QueryRunner queryRunner = getQueryRunner();
+        String table = "test_customer_union";
+        String view = "test_customer_view_union";
+        try {
+            computeActual(format("CREATE TABLE %s WITH (partitioned_by = ARRAY['nationkey']) " +
+                    "AS SELECT custkey, name, address, nationkey FROM customer", table));
+
+            String baseQuery = format(
+                    "SELECT name, custkey, nationkey FROM %s WHERE custkey < 1000 EXCEPT " +
+                            "SELECT name, custkey, nationkey FROM %s WHERE custkey > 900", table, table);
+
+            assertUpdate(format("CREATE MATERIALIZED VIEW %s WITH (partitioned_by = ARRAY['nationkey']) " +
+                    "AS %s", view, baseQuery));
+
+            assertUpdate(format("REFRESH MATERIALIZED VIEW %s WHERE nationkey < 10", view, baseQuery), 380);
```

Comment:
Here only one argument is expected for the format string but found two.

Reply:
Forgot to change this?